api_gateway/src/main/kotlin/org/thechance/api_gateway/endpoints/CuisineRoute.kt

```diff
@@ -25,6 +26,15 @@ fun Route.cuisineRoute() {
                 val cuisine = restaurantService.addCuisine(name, language)
                 respondWithResult(HttpStatusCode.Created, cuisine)
             }
+            delete("/{id}") {
+                val language = extractLocalizationHeader()
+                val params = call.receiveParameters()
```

Comment:
It will take id as form data
Please make it just call.parametrs["id"] 
Which take from path

Reply:
> It will take id as form data
> Please make it just call.parametrs["id"] 
> Which take from path

It will solve this issue https://github.com/TheChance101/beep-beep/pull/330#pullrequestreview-1612522904